### ISSUE TITLE
vokoscreen not dumping screen material to a file

### ISSUE THREAD

**Issue by drozalski** [vkohaupt/vokoscreen#202]
My system specs are linux mint 18.2 'Sonya' 64 bit,
and the issue is that when the capturing process is over 
after clicking stop button the captured video is not dumped to the
file on the disc and no file is written. 
After screen capturing is over, when going to the destination directory, the file is not there.
But when the directory is remaining opened in a directory window 
the next screen capture is written to a new file 
even after the previous screen capture hasn't been written to a file.
So to actually capture anything to a file 
I have to 'test' vokoscreen on some irrelevant content 
to check if the file is not going to be lost 
and only then the next capture can be started and finalized with an actual video file.

Please say if it is a failure on my side or is the the vokoscreen failure,
because this way I lost some important material and time.
I'm looking forward to seeing if this issue is resolved.
Thanks in advance, Daniel Rozalski.

**Comment by tuxmuck** [comment]
There are no known problems with this -

How long (time) was the missing recording?
Recorded with mpeg4 or libx264?
What processor do you have?

**Comment by drozalski** [comment]
thanks for reply
the problem got resolved by upgrading to vokoscreen 2.5
the problem was with vokoscreen 2.4
the processor is core 2 duo e6750
the recording was 1h:20m
codec was libx264

**Comment by vkohaupt** [comment]
You take avi with pause?
Avi is a very old and with pause a very bad format.
Please take mkv.

**Comment by drozalski** [comment]
Yes it was avi but hadn't been pausing though.
I'll try now with mkv.
Thanks

**Comment by drozalski** [comment]
sorry to say this but it happened again. 
I was using mkv and it was about 20 minutes of recording and the file hasn't been created.
after this i was shocked and tried to record anything and the short file about 20 seconds was created
but the long one hasn't been. I haven't  been pausing. Is this God's will at work that catching fast recordings for long is different from making a test? Now I might believe it but it was supposed to be a recording of a tale for kids from a popular channel. Now it might not be a technical thing but the kid
 is not going to watch her story. Thanks for your comments.

**Comment by drozalski** [comment]
Solved. The problem was not with vokoscreen. it was with the destination for dumping a file.
It has to be mounted while dumping file and it was not mounted when I was trying to make the
recording. When I opened the destination folder the volume was being mounted and when the
folder was not opened the destination volume was not mounted so the file was dumped only when i first mounted the volume. Sorry for my English not being strong and I wish to thank you for your timely replies.
